mldsa/sign.c

```diff
@@ -14,6 +14,81 @@
 #include "sign.h"
 #include "symmetric.h"
 
+
+static int mld_check_pct(uint8_t const pk[CRYPTO_PUBLICKEYBYTES],
```

Comment:
Is there a good reason why mld_check_pct() needs a distinct declaration/contract from its definition below? Can't they just be merged?